sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -1135,6 +1135,20 @@ def _add_argparse_args(cls, parser):
         dest='min_cpu_platform',
         type=str,
         help='GCE minimum CPU platform. Default is determined by GCP.')
+    parser.add_argument(
+        '--max_cache_memory_usage_mb',
+        dest='max_cache_memory_usage_mb',
+        type=int,
+        default=100,
+        help=(
+            'Size of the SdkHarness cache to store user state and side inputs '
+            'in MB. Default is 100MB. If the cache is full, least recently '
+            'used elements will be evicted. This cache will be per '
+            'SdkHarness/Sdk Process. SDKHarness is a python process,'
```

Comment:
```suggestion
            'SdkHarness. SDKHarness is a python process,'
```

Optional, but I think this is clearer since we then explain what an SDK harness is right after this.